Assets/Unity Simple Liquid/Scripts/SplitController.cs

```diff
@@ -312,6 +313,17 @@ private RaycastHit FindLiquidContainer(Vector3 splitPos, GameObject ignoreCollis
 
 		#endregion
 
+		#region ChangeColor
+		private void SendLiquidContainer(GameObject go)
+		{
+			LiquidContainer lc = go.GetComponent<LiquidContainer>();
+			if (!lc)
+				return;
```

Comment:
It's better not GetComponent from go each frame, but pass LiquidContainer as parameter. Safe some performance.

Reply:
yes it turned out there is such a possibility and this is the best solution